expandybird/expansion/sandbox_loader.py

```diff
@@ -95,7 +95,7 @@ def process_imports(imports):
 
   # Now build the hierarchical modules.
   for k in imports.keys():
-    if imports[k].endswith('.jinja'):
+    if imports[k]['path'].endswith('.jinja'):
       continue
     # Normalize paths and trim .py extension, if any.
     normalized = os.path.splitext(os.path.normpath(k))[0]
```

Comment:
My guess is that you will need to have this operate on path rather than k, as it's expecting to normalize the python file name